webpack.config.js

```diff
@@ -219,19 +113,11 @@ module.exports = {
         splitChunks: {
             cacheGroups: {
                 commons: {
-                    test: /[\\/]node_modules[\\/]/,
+                    test: /[\\/]node_modules[\\/].+\.(js|ts)$/, // split js only
                     name: 'vendors',
-                    chunks (chunk) {
-                        // exclude `style`
-                        return chunk.name !== 'style';
```

Comment:
you can specify source SCSS file directly in HTML, not in a JS file,
therefore this option is not needed